### ISSUE TITLE
Action Bar: bar action item group doesn't have property to point to child it controls

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#712]
### Expected behavior

Button element that has aria-expanded attribute should have aria-controls attribute pointing to the id of the container that it controls and has expanded state attribute for. The container is not a child of the button element so this pointer creates that programatic connection. 
### Actual behavior

Missing aria-controls attributes.

Technique is described in bootstrap documentation - http://v4-alpha.getbootstrap.com/components/collapse/ (Accessibility section in middle of page)

**Comment by Blackbaud-PatrickOFriel** [comment]
Looks like ui-bootstrap associates the dropdown with the toggle using the `aria-labelledby` which is another option. See https://angular-ui.github.io/bootstrap/#/dropdown

**Comment by Blackbaud-MattGregg** [comment]
aria-labelledby could be added as well, but this is for a different purpose. It provides information about the menu. aria-controls creates the programmatic relationship about the button controlling the menu which is part of authoring practice for using expand/collapse states - https://www.w3.org/TR/wai-aria-1.1/#aria-expanded 

A short article on this - https://www.marcozehe.de/2010/02/10/easy-aria-tip-5-aria-expanded-and-aria-controls/

**Comment by Blackbaud-PatrickOFriel** [comment]
Ah gotcha, makes sense

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.6.8](https://github.com/blackbaud/skyux/releases/tag/v1.6.8)
